src/redux/dialog-window/dialog-window.reducer.js

```diff
@@ -4,7 +4,7 @@ import { CLOSE_DIALOG, SHOW_DIALOG } from './dialog-window.types';
 export const basicSelector = ({ DialogWindow }) => ({
   isOpen: DialogWindow.isOpen,
   dialogTitle: DialogWindow.dialogTitle,
-  dialogContent: DialogWindow.dialogContent,
+  dialogContent: DialogWindow.content,
```

Comment:
You changed this but in initialState we still have dialogContent

Reply:
Fixed.

Reply:
Fixed.